pkg/dbclient/client.go

```diff
@@ -567,24 +567,42 @@ func (pc *client) UserExists(userName string) (bool, error) {
 	return exists, nil
 }
 
-func (pc *client) DeleteUser(username, reassignToUser string) error {
-	db := pc.DB
+func (pc *client) DeleteUser(dbName, username, reassignToUser string) error {
 	if username == "" {
 		err := fmt.Errorf("an empty username")
 		pc.log.Error(err, "error occurred")
 		metrics.UsersDeleted.Inc()
 		return err
 	}
 
-	pc.log.Info("drop user", "user:", username)
-	_, err := db.Exec(fmt.Sprintf(
-		`REASSIGN OWNED BY %s TO %s;
-DROP USER %s;`, pq.QuoteIdentifier(username), pq.QuoteIdentifier(reassignToUser), pq.QuoteIdentifier(username)))
+	var exists bool
+
+	err := pc.DB.QueryRow("SELECT EXISTS(SELECT pg_user.usename FROM pg_catalog.pg_user where pg_user.usename = $1)", username).Scan(&exists)
```

Comment:
All of these calls have context variations
```suggestion
	err := pc.DB.QueryRowContext(ctx, "SELECT EXISTS(SELECT pg_user.usename FROM pg_catalog.pg_user where pg_user.usename = $1)", username).Scan(&exists)
```